opencxl/bin/single_logical_device.py

```diff
@@ -24,10 +24,7 @@ async def run_devices(slds: List[SingleLogicalDevice]):
     try:
         await asyncio.gather(*(sld.run() for sld in slds))
     except Exception as e:
-        logger.error(
-            "An error occurred while running the Single Logical Device clients.",
-            exc_info=e,
-        )
+        logger.error("Error while running Single Logical Device Device", exc_info=e)
```

Comment:
My only complaint is that maybe this is confusingly worded---maybe:
`Error while running "Single Logical Device" device`
would be more clear?

Reply:
That was a typo. Fixed.

Also, don't approve unless I (or the author of PR) addresses all of your concerns.